draft-ietf-rtcweb-jsep.xml

```diff
@@ -3174,17 +3164,63 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           processing MUST stop and an error MUST be returned.</t>
 
           <t>Next, the SessionDescription is parsed into a data
-          structure, as described in the
-          <xref target="sec.parsing-a-desc" /> section below. If parsing
+          structure, as described in
+          <xref target="sec.parsing-a-desc" /> below. If parsing
           fails for any reason, processing MUST stop and an error MUST
           be returned.</t>
 
           <t>Finally, the parsed SessionDescription is applied as
-          described in the
-          <xref target="sec.applying-a-remote-desc" /> section
+          described in
+          <xref target="sec.applying-a-remote-desc" />
           below.</t>
         </list></t>
       </section>
+
+      <section title="Processing a Rollback"
+               anchor="sec.processing-a-rollback">
+
+        <t>  The rollback is used to recover from possible errors and
+        return to stable state the system was in before a new offer /
+        answer negotiation started. </t>
+
+        <t> When either setLocalDescription or setRemoteDescription
+        method is called with a a session description of type
+        "rollback" a rollback occurs. </t>
+
+        <t> If the system is currently in the "stable" state, no
+        further processing happens. </t>
+
+        <t>A rollback MUST discard any proposed changes to the session,
+        returning the state machine to the stable state, and setting
+        the pending local and/or remote description (see <xref
+        target="sec.pendinglocaldescription" /> and <xref
+        target="sec.pendingremotedescription" />) to null.  Any
+        resources or candidates that were allocated by the abandoned
+        local description are discarded; any media that is received
+        will be processed according to the previous local and remote
```

Comment:
Note, the text about whether a pranswer is the current description is vague

"   The pendingLocalDescription method returns a copy of the local
   description currently in negotiation - i.e., a local offer set
   without any corresponding remote answer - in addition to any local
   candidates that have been generated by the ICE agent since the local
   description was set.
"